search-service/src/main/kotlin/com/egm/stellio/search/service/TemporalEntityAttributeService.kt

```diff
@@ -275,6 +276,28 @@ class TemporalEntityAttributeService(
             .collectList()
     }
 
+    fun getTemporalsCount(ids: Set<URI>, types: Set<String>, attrs: Set<String>): Mono<Int> {
+        var selectStatement =
+            """
+            SELECT count(*) from temporal_entity_attribute
```

Comment:
there is a trap here: an entity will typically have more than one entry in the `temporal_entity_attribute` (actually, it will have one per temporal attribute), so you'll just want the count of distinct entities.